gmail/style.css

```diff
@@ -971,7 +971,7 @@ html.simpl .aSs > .aSt {
 	left: calc(50vw - 450px) !important;
 }
 html.simpl .aSs .aDg > .aDj {
-	width: 865px !important;
+	width: 867px !important;
```

Comment:
Why does this need to be changed (not that it matters a ton)?

Reply:
This is done to fix the scrollbar appearing beneath the formatting bar.

![](https://user-images.githubusercontent.com/24394958/57516684-cc05c500-7332-11e9-92df-d2eec7f64b8b.png)